api/src/migrations/1543494409127-change-translation-value-type.ts

```
@@ -70,7 +71,7 @@ export class changeTranslationValueType1543494409127 implements MigrationInterfa
         console.log('No column type change needed for SQLite down migration');
         break;
       default:
-        console.log('Unknown DB type');
+        throw new Error('Unknown DB type: ' + config.db.default.type);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)